test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class CommentsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Nice to have tests coverage.

Reply:
@violeta-p I have not yet implemented Rspec.
But I will do it on the next PR, if it's not a problem?